roqoqo/src/operations/bosonic_operations.rs

```diff
@@ -22,7 +22,7 @@ use qoqo_calculator::CalculatorFloat;
 
 /// The single-mode squeezing gate with tunable squeezing.
 ///
-/// The squeezing gate is a quantum operation that allows for precise manipulation of quantum states,
+/// The squeezing gate is a quantum operation that reduces that allows for precise manipulation of quantum states,
```

Comment:
typo?

Reply:
Shoot this should have been fixed in pulling, thank you